app/views/govuk_publishing_components/components/_feedback.html.erb

```diff
@@ -0,0 +1,112 @@
+    <a href="#" class="pub-c-feedback__close pub-c-feedback__js-show js-close-form" aria-controls="something-is-wrong" aria-expanded="true" role="button">Close</a>
+
+    <div class="pub-c-feedback__grid-row">
+      <div class="pub-c-feedback__column-two-thirds">
+        <div class="pub-c-feedback__error-summary pub-c-feedback__js-show js-hidden js-errors" tabindex="-1"></div>
+
+        <input type="hidden" name="url" value="<%= request.original_url -%>">
+        <input type="hidden" name="user_agent" value="<%= request.user_agent -%>">
+
+        <h2 class="pub-c-feedback__form-heading">Help us improve GOV.UK</h2>
+        <p id="feedback_explanation" class="pub-c-feedback__form-paragraph">Don’t include personal or financial information like your National Insurance number or credit card details.</p>
+
+        <%= render "govuk_publishing_components/components/input", {
+          label: {
+            text: "What were you doing?"
+          },
+          name: "what_doing"
+        } %>
+
+        <%= render "govuk_publishing_components/components/input", {
+          label: {
+            text: "What went wrong?"
+          },
+          name: "what_wrong"
+        } %>
+
+        <input class="pub-c-feedback__submit" type="submit" value="Send">
+      </div>
+    </div>
+  </form>
+
+  <form action="/contact/govuk/email-survey-signup" id="page-is-not-useful" class="pub-c-feedback__form pub-c-feedback__form--email pub-c-feedback__js-show js-feedback-form js-hidden" data-track-category="user_satisfaction_survey" data-track-action="banner_taken">
+    <a href="#" class="pub-c-feedback__close js-close-form" aria-controls="page-is-not-useful" aria-expanded="true" role="button">Close</a>
+
+    <div class="pub-c-feedback__grid-row">
+      <div class="pub-c-feedback__column-two-thirds">
+        <div class="pub-c-feedback__error-summary js-hidden js-errors" tabindex="-1"></div>
+
+        <input type="hidden" name="url" value="<%= request.original_url -%>">
+        <input type="hidden" name="user_agent" value="<%= request.user_agent -%>">
+
+        <input name="email_survey_signup[survey_id]" type="hidden" value="user_satisfaction_survey">
+        <input name="email_survey_signup[survey_source]" type="hidden" value="<%= request.original_url -%>">
+        <input name="email_survey_signup[ga_client_id]" type="hidden" value="1627485790.1515403243">
+
+        <h2 class="pub-c-feedback__form-heading">Help us improve GOV.UK</h2>
+        <p id="survey_explanation" class="pub-c-feedback__form-paragraph">To help us improve GOV.UK, we’d like to know more about your visit today. We’ll send you a link to a feedback form. It will take only 2 minutes to fill in. Don’t worry we won’t send you spam or share your email address with anyone.</p>
+
+        <%= render "govuk_publishing_components/components/input", {
+          label: {
+            text: "Email address"
+          },
+          name: "email_survey_signup[email_address]"
```

Comment:
We can use `email` type now: https://github.com/alphagov/govuk_publishing_components/pull/165

(Thanks @kevindew)